encord/objects/bitmask.py

```diff
@@ -118,16 +118,21 @@ def __init__(self, source: Any):
 
         if isinstance(source, BitmaskCoordinates.EncodedBitmask):
             self._encoded_bitmask = source
+        elif (
+            isinstance(source, dict)
+            and len(set(source.keys()).intersection({"top", "left", "height", "width", "rle_string"})) == 5
```

Comment:
Perhaps this check would be more clear using runtime checkable protocols? That would also provide better type checking